crates/ark/src/interface.rs

```diff
@@ -230,8 +228,8 @@ pub struct RMain {
 struct ActiveReadConsoleRequest {
     exec_count: u32,
     request: ExecuteRequest,
-    originator: Originator,
-    response_tx: Sender<ExecuteResponse>,
+    orig: Originator,
```

Comment:
I actually purposefully made this `originator` everywhere to avoid shorthand notation